ale/trainer/prediction_result.py

```diff
@@ -24,7 +24,7 @@ class TokenConfidence(BaseModel):
     text: str
     label_confidence: List[LabelConfidence]
     gold_label: Optional[str] = None
-    predicted_label: str
+    predicted_label: str = ""
```

Comment:
or as optional with None. But each Prediction should have a predicted_label. What is the case?

Reply:
I was not sure if it is always "0" or if it can also be left empty if no named entity is predicted.

Reply:
There should be an O Tag. Otherwise, pydantic would raise an exception due to a missing field. Did you face an exception?

Reply:
Yes, if I test it, I only want to pass the token string and the confidences. If we don't set a default here, we get an exception that the value for predicted label is missing. I do not want to pass it explicitly because the information is already given by the confidences. Instead of setting a default and using the get_predicted_label function, we could also directly set predicted_label when initializing the object?

Reply:
I introduced the field because, using the CRF method, the predicted label is not always the label with the highest confidence.